uelc/main/views.py

```diff
@@ -29,6 +29,14 @@ def dispatch(self, *args, **kwargs):
         return super(LoggedInMixinSuperuser, self).dispatch(*args, **kwargs)
 
 
+class LoggedInMixinFacilitator(object):
```

Comment:
`LoggedInMixinFacilitator` sounds weird to me. Shouldn't it be `LoggedInFacilitatorMixin` instead?